x/marker/keeper/marker.go

```diff
@@ -673,7 +673,7 @@ func (k Keeper) TransferCoin(ctx sdk.Context, from, to, admin sdk.AccAddress, am
 	if !m.AddressHasAccess(admin, types.Access_Transfer) {
```

Comment:
This specific check is why the forced transfer/clawback only applies to restricted coins ... because regular coins can not have this grant created.